### PR TITLE
feat: Dispatch dismissed event in common snackbar - EXO-67599 - Meeds-io/meeds#1367 (#3249)

### PR DESCRIPTION
**Pull request by Jihed525**
Current common snackbar doesn't satisfy some functional needs for specific applications. The non existing of a dispatched event on alert dismiss prevents triggering actions on some app side on the dismiss state.
This PR makes sure to dispatch a dismissed event on auto dismiss or a dismiss by close button of the alert.

(cherry picked from commit 6bfe6abce5f5533e6286b0c7449f7aceb1c87509)

### CODE REVIEW

File: webapp/portlet/src/main/webapp/vue-apps/common/components/Notifications.vue [hunk 1]
```diff
@@ -92,7 +93,7 @@
       <template v-if="!isMobile" #close="{toggle}">
         <v-btn
           icon
-          @click="toggle">
+          @click="dispatchDismissed(!toggle)">
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/common/components/Notifications.vue:96]: The `toggle` is a function that must be called to toggle alert closing as well.

Comment (boubaker) [reply webapp/portlet/src/main/webapp/vue-apps/common/components/Notifications.vue:96]: Never-mind, will fix it when needed.

Comment (ahamdi) [reply webapp/portlet/src/main/webapp/vue-apps/common/components/Notifications.vue:96]: @boubaker Could you five me more details, I will check it with the developers and fix it.

**Review by ahamdi — APPROVED** [review]

File: webapp/portlet/src/main/webapp/vue-apps/common/components/Notifications.vue (hunk 1 above)
Comment (boubaker) [reply webapp/portlet/src/main/webapp/vue-apps/common/components/Notifications.vue:96]: > @boubaker Could you five me more details, I will check it with the developers and fix it.

Yes, if you want, but it's for better usage of Vue & Vuetify builtin functions.
As said, the `toggle` is a function that will be used to update the `v-model` of `v-alert` (as you can see [it here](https://v2.vuetifyjs.com/en/api/v-alert/#slots)). In order to drive the v-alert hide, we can rely on `@input` event of `v-alert` or add a dedicated `v-model` for the `v-alert` and thus a `watch`ed `data` variable.

This is just to ensure that the events are triggered when clicking on button or elsewhere, even when the alert is closed due to timeout. So this is just an enhancement and not blocker, that's why it was marked as resolved.

But, even with the current change, knowing that `toggle` variable is never `falsy`, `dispatchDismissed(!toggle)` is exactly the same as `dispatchDismissed(false)`.

To not have to retest other changes, knowing that all of this are enhancements, no need IMO to apply further change. We will change it `when needed` as said :+1:

**Pull request merged**